src/olympia/addons/models.py

```diff
@@ -637,10 +637,12 @@ def clean_slug(self, slug_field='slug'):
 
         clean_slug(self, slug_field)
 
-    def force_disable(self):
+    def force_disable(self, user=None):
         from olympia.reviewers.models import NeedsHumanReview
 
-        activity.log_create(amo.LOG.FORCE_DISABLE, self)
+        activity.log_create(
+            amo.LOG.FORCE_DISABLE, self, **({'user': user} if user else {})
+        )
```

Comment:
This is inconsistent with other reviewer actions, we should be recording the `reason` and `details` (which includes `comments`) in that activity.